CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,3 @@
-- Hadoop extensions for CKAN (#1)
+- [tidoop-hadoop-ext] [FEATURE] Hadoop extensions for CKAN (read mode) (#1)
+- [tidoop-hadoop-ext] [FEATURE] Hadoop extensions for CKAN (write mode) (#6)
+- [tidoop-mr-lib] [FEATURE] Add Filter MR job (#15)
```

Comment:
The issue #1 has been renamed, since the concept of `tidoop-hadoop-ext`, `tidoop-mr-lib` etc subfolders did not exist when #1 was implemented. Regarding the issue #6, well, that change was not reflected in `CHANGES_NEXT_RELEASE` when such an issue was implemented... so it has been added right now :D